src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+import java.io.IOException;
+
```

Comment:
Perhaps you want to add more descriptive header comments for this class and the public methods in it? I think the comments are missing for some other classes as well (e.g. DukeException and Ui classes)